vipermonkey/core/__init__.py

```diff
@@ -140,13 +142,24 @@ class ViperMonkey(object):
     # TODO: load multiple modules from a file using olevba
 
     def __init__(self, filename):
+        self.metadata = None
         self.filename = filename
         self.modules = []
         self.modules_code = []
         self.globals = {}
+        self.externals = {}
         # list of actions (stored as tuples by report_action)
         self.actions = []
 
+        # Figure out whether this is VBScript or VBA.
+        file_info = subprocess.check_output(["file", self.filename]).strip()
```

Comment:
@kirk-sayre-work I will need to change this, vmonkey should not call file as it's not available on all platforms. Python and olefile/oletools should already provide equivalent features to check file types. I will fix it on the main repo.

Reply:
I've added a direct magic # check and eliminated the execution of the file utility in my dev fork.